click/termui.py

```diff
@@ -177,7 +177,11 @@ def get_terminal_size():
             return sz.columns, sz.lines
 
     if get_winterm_size is not None:
-        return get_winterm_size()
+        size = get_winterm_size()
+        if size == (0, 0):
```

Comment:
Can you add an inline comment that reflect the same info as the commit message?

Thanks